### DISCUSSION TITLE
`select` and `multi-select` filterVariants' tooltip could match label instead of value

### DISCUSSION THREAD

**Discussion by loughlinclaus3** [KevinVandy/material-react-table#994] [Ideas]
So, when a user is filtering a column, there's a `FilterAltIcon` next to the column header. And when the user hovers their mouse over it, a tooltip pops up, saying "filtering by {header} - {filterFn} {filterValue}". This filter tooltip is [defined in the code here](https://github.com/KevinVandy/material-react-table/blob/1a2587edcc9be2856c7f99014ff5f3a484c2e3c1/packages/material-react-table/src/components/head/MRT_TableHeadCellFilterLabel.tsx#L52).

Now, if the column's `filterVariant` is set to either `select` or `multi-select`, users can also choose to provide an array of either strings or `{label: string, value: string, text:string}` (note: `text` is deprecated) as `filterSelectOptions` (assuming that the `enableFacetedValues` is false). In the scenario that a user provides an array of `{label: string, value: string, text:string}`, the tooltip displays the filterValue's `value`, instead of the associated `label`. I think that the tooltip would be improved if it showed the `label` instead of the `value`. What are your thoughts on this?

Example in current version:
<img width="186" alt="Screenshot 2024-02-16 at 2 57 09 PM" src="https://github.com/KevinVandy/material-react-table/assets/108533329/83b1cac5-9a74-40d6-bde8-107e0fbae1ba">

Example of desired update:
<img width="207" alt="Screenshot 2024-02-16 at 2 58 20 PM" src="https://github.com/KevinVandy/material-react-table/assets/108533329/ccdd6df8-91b7-4cef-aafd-3696b60f7e1c">

**Comment by KevinVandy** [comment]
Sure, make a PR if you want

**Reply by loughlinclaus3** [reply]
Will do
